src/openzeppelin/account/account.cairo

```diff
@@ -88,15 +113,21 @@ mod Account {
             validate_transaction()
         }
 
-        fn is_valid_signature(message: felt252, signature: Array<felt252>) -> felt252 {
-            if _is_valid_signature(message, signature.span()) {
+        fn is_valid_signature(hash: felt252, signature: Array<felt252>) -> felt252 {
+            if _is_valid_signature(hash, signature.span()) {
                 starknet::VALIDATED
             } else {
                 0
             }
         }
     }
 
+    impl SRC6CamelImpl of ISRC6Camel {
+        fn isValidSignature(hash: felt252, signature: Array<felt252>) -> felt252 {
+            SRC6Impl::is_valid_signature(hash, signature)
+        }
```

Comment:
We are now implementing SRC6 in Account (after some discussion with other accountors), and the standard is to return `VALID`. Both interfaces must return the same value. The standard also suggests that until regenesis at least, dapps should check for bool as well when calling is valid signature for backward compatibility.

Maybe we should return VALID when the inner call returns bool in the account dual dispatcher as well? I think this can be easily implemented in wallets or dapps, IMO we shouldn't.

Reply:
> Both interfaces must return the same value

Ahh right, I misunderstood. Yeah, I see this being much more easily solved in wallets/dapps as we establish the returning `VALID` standard